nav2_dwb_controller/dwb_core/include/dwb_core/exceptions.hpp

```diff
@@ -38,16 +38,10 @@
 #include <string>
 #include <memory>
 
-namespace nav_core2
-{
+#include "nav2_core/exceptions.hpp"
 
-class PlannerException : public std::runtime_error
+namespace nav2_core
```

Comment:
Should we make this `dwb_core` instead of `nav2_core`?

Reply:
Yes, will do that.